pkg/vulnsrc/redhat-oval/types/types.go

```diff
@@ -1,65 +1,65 @@
-package redhatoval
+package types
```

Comment:
Could you tell me the reason why we want to move this package?

Reply:
we need to decode [Advisory](https://github.com/aquasecurity/trivy-db/blob/2c0339bb908551e6a6eb5fe5b22a4a004ce42333/pkg/vulnsrc/redhat-oval/types.go#L151-L176) structure here: https://github.com/aquasecurity/trivy-db/blob/03e5c844a77f00303a550476af124d753616575e/pkg/db/advisory_detail.go#L119-L128
this is necessary to remove rejected cves from `CveEntry`.
But we can't import `redhatoval` package from `db` package, because we will get `cycle import`.(`redhatoval` imported `db` for `PutAdvisoryDetail` func)